dev/com.ibm.ws.artifact.zip/src/com/ibm/ws/artifact/zip/cache/internal/ZipFileHandleImpl.java

```diff
@@ -88,7 +88,10 @@ public long getLastModified() {
 
     //
 
-    private final Integer zipFileLock = new Integer(0);
+    private class ZipFileLock {
```

Comment:
could this be made a static class to reduce overhead of creating a new `ZipFileHandleImpl` (same comment applies to other instances of occurrences of non-static private lock classes)

Reply:
I just put it back to what it was before, but sure, it makes sense to make them all static.